app/controllers/users/invitations_controller.rb

```diff
@@ -38,6 +47,14 @@ def invited_user_already_exists?
     !!User.find_by(email: invite_params[:email])
   end
 
+  def invited_user_not_confirmed?
+    User.find_by(email: invite_params[:email]).confirmed_at.nil?
```

Comment:
`.confirmed?` would be a better method to use here: https://github.com/plataformatec/devise/blob/master/lib/devise/models/confirmable.rb#L106

eg: `!User.find.....confirmed?`
```suggestion
    User.find_by(email: invite_params[:email]).confirmed_at.nil?
```